### ISSUE TITLE
word2vec legal model

### ISSUE THREAD

**Issue by ShrikanthSingh** [LexPredict/lexpredict-lexnlp#43]
As stated in the paper the pre-trained models will be available from Oct 2017, but I could not find any of them in the repo. Can someone help me out it locating it.

**Comment by vi-adm** [comment]
@ShrikanthSingh 
Same question here. Have you figured it out?

**Comment by ShrikanthSingh** [comment]
No, there wasn't any reply from the contributors. I even mailed them via the contact given in the published paper. But no response. So I just built my own word2vec with the dataset I have.

**Comment by bytearchive** [comment]
@ShrikanthSingh what dataset did you use? is it available openly?

**Comment by esvhd** [comment]
I also saw in [this](https://www.lexpredict.com/2018/06/openedgar-open-source-software-sec-edgar-database/) article that pre-trained word2vec model would be available here in this repo. 

But reading this thread it sounds like it's not present here? It's not obvious reading the docs page where it could be...
